experimental/hip/native_executable.h

```diff
@@ -41,10 +41,10 @@ iree_status_t iree_hal_hip_native_executable_create(
 // |executable|.
 iree_status_t iree_hal_hip_native_executable_entry_point_kernel_params(
     iree_hal_executable_t* executable, int32_t entry_point,
-    iree_hal_hip_kernel_params_t* out_params);
+    iree_hal_hip_kernel_info_t* out_params);
```

Comment:
Rename this function to `_kernel_info` and paramerter to `out_info` to be consistent. (Should also double check other places we use `_params`.